components/match2/wikis/wildrift/match_summary.lua

```diff
@@ -0,0 +1,364 @@
+local Table = require('Module:Table')
+local ExternalLinks = require('Module:ExternalLinks')
+local String = require('Module:StringUtils')
+
+local MatchGroupUtil = Lua.import('Module:MatchGroup/Util', {requireDevIfEnabled = true})
+local MatchSummary = Lua.import('Module:MatchSummary/Base', {requireDevIfEnabled = true})
+local OpponentDisplay = Lua.import('Module:OpponentDisplay', {requireDevIfEnabled = true})
+
+local _MAX_NUM_BANS = 5
+local _NUM_CHAMPIONS_PICK = 5
+
+local _GREEN_CHECK = '[[File:GreenCheck.png|14x14px|link=]]'
+local _NO_CHECK = '[[File:NoCheck.png|link=]]'
+
+local _EPOCH_TIME_EXTENDED = '1970-01-01T00:00:00+00:00'
+
+-- Champion Ban Class
+local ChampionBan = Class.new(
+	function(self)
+		self.root = mw.html.create('div'):addClass('brkts-popup-mapveto')
+		self.table = self.root:tag('table')
+			:addClass('wikitable-striped'):addClass('collapsible'):addClass('collapsed')
+		self:createHeader()
+	end
+)
+
+function ChampionBan:createHeader()
+	self.table:tag('tr')
+		:tag('th'):css('width','40%'):wikitext(''):done()
+		:tag('th'):css('width','20%'):wikitext('Bans'):done()
+		:tag('th'):css('width','40%'):wikitext(''):done()
+	return self
+end
+
+function ChampionBan:banRow(banData, gameNumber, numberOfBans)
+	self.table:tag('tr')
+		:tag('td')
+			:node(CustomMatchSummary._opponentChampionsDisplay(banData[1], numberOfBans, false, true))
+		:tag('td')
+			:node(mw.html.create('div')
+				:wikitext(CustomMatchSummary._createAbbreviation{
+					title = 'Bans in game ' .. gameNumber,
+					text = 'Game ' .. gameNumber,
+				})
+			)
+		:tag('td')
+			:node(CustomMatchSummary._opponentChampionsDisplay(banData[2], numberOfBans, true, true))
+	return self
+end
+
+function ChampionBan:create()
+	return self.root
+end
+
+-- MVP Class
+local MVP = Class.new(
+	function(self)
+		self.root = mw.html.create('div'):addClass('brkts-popup-footer'):addClass('brkts-popup-mvp')
+		self.players = {}
+	end
+)
+
+function MVP:addPlayer(player)
+	if not Logic.isEmpty(player) then
+		table.insert(self.players, player)
+	end
+	return self
+end
+
+function MVP:create()
+	local span = mw.html.create('span')
+	span:wikitext(#self.players > 1 and 'MVPs: ' or 'MVP: ')
+	for index, player in ipairs(self.players) do
+		if index > 1 then
+			span:wikitext(', ')
+		end
+		span:wikitext('[['..player..']]')
+	end
+	self.root:node(span)
+	return self.root
+end
+
+
+function CustomMatchSummary.getByMatchId(args)
+	local match = MatchGroupUtil.fetchMatchForBracketDisplay(args.bracketId, args.matchId)
+
+	local matchSummary = MatchSummary():init('420px')
+	matchSummary.root:css('flex-wrap', 'unset') -- temporary workaround to fix height, taken from RL
+
+	matchSummary:header(CustomMatchSummary._createHeader(match))
+				:body(CustomMatchSummary._createBody(match))
+
+	if match.comment then
+		local comment = MatchSummary.Comment():content(match.comment)
+		comment.root:css('display', 'block'):css('text-align', 'center')
```

Comment:
Should be moved to MatchSummary/Base

Reply:
okay for me

Reply:
alternative would be to change the css sheet

Reply:
Fine by me to change the css